beaver_store/lib/src/storage_service/gcloud_storage_service.dart

```diff
@@ -102,4 +111,24 @@ class GCloudStorageService implements StorageService {
   Future<bool> setBuildNumber(String projectId, int buildNumber) {
     // TODO: implement setBuildNumber
   }
+
+  String _getDefaultProjectName() {
+    final projectName = Platform.environment['GCLOUD_PROJECT_NAME'];
```

Comment:
On the trigger handler side, currently, project is obtained through BeaverStore.getProject. It's deadlock since project and BeaverStore requires each other. I want to put it off until that cloud_project_name is removed from beaver.yaml.